style_editor.html

```diff
@@ -1476,6 +1476,7 @@
 
   width = window.innerWidth;
   height = window.innerHeight;
+  canvas_id.setAttribute("title", "Blade Preview.\nMove mouse to swing. Click to Clash\nor to Do Selected Effect (and to dismiss this Tooltip.)\nGoto settings to change hilt model or toggle Mouse Swings mode (swinging with mouse moves.)");
```

Comment:
Why do this in JS rather than HTML?

Reply:
Have to jog the memory a bit.
I think it had to do with the swapping of title and data-title to turn them off.

Reply:
```js
var tipsState = new SavedState("tips", true, (on) => { 
 if (on) {
    const elementsWithDataTitles = document.querySelectorAll("[data-title]");
    elementsWithDataTitles.forEach((element) => {
      element.setAttribute("title", element.dataset.title);
      element.removeAttribute("data-title");
    });
  } else {
    const elementsWithTitles = document.querySelectorAll("[title]");
    elementsWithTitles.forEach((element) => {
      element.dataset.title = element.getAttribute("title");
      element.removeAttribute("title");
    });
  }
});
```